README.md

```diff
@@ -36,22 +25,28 @@ learning frameworks.
 |[Sparsity](./docs/compression_algorithms/Sparsity.md) | Supported | Supported |
 |[Filter pruning](./docs/compression_algorithms/Pruning.md) | Supported | Supported |
 
-- Automatic, configurable model graph transformation to obtain the compressed model.
-  > **NOTE**: Limited support for TensorFlow models. The models created using Sequential or Keras Functional API are only supported.
-- Common interface for compression methods.
-- GPU-accelerated layers for faster compressed model fine-tuning.
-- Distributed training support.
-- Configuration file examples for each supported compression algorithm.
-- Git patches for prominent third-party repositories ([huggingface-transformers](https://github.com/huggingface/transformers)) demonstrating the process of integrating NNCF into custom training pipelines
-- Exporting PyTorch compressed models to ONNX\* checkpoints and TensorFlow compressed models to SavedModel or Frozen Graph format, ready to use with [OpenVINO&trade; toolkit](https://github.com/openvinotoolkit/).
-- Support for [Accuracy-Aware model training](./docs/Usage.md#accuracy-aware-model-training) pipelines via the [Adaptive Compression Level Training](./docs/accuracy_aware_model_training/AdaptiveCompressionLevelTraining.md) and [Early Exit Training](./docs/accuracy_aware_model_training/EarlyExitTraining.md).
+All of these algorithms can be executed through [Accuracy-Aware model training](./docs/Usage.md#accuracy-aware-model-training) pipelines: [Adaptive Compression Level Training](./docs/accuracy_aware_model_training/AdaptiveCompressionLevelTraining.md) and [Early Exit Training](./docs/accuracy_aware_model_training/EarlyExitTraining.md). 
+They allow to achieve the maximum compression level or to reduce the fine-tuning time, while staying within the range of the user-defined maximum accuracy degradation.
+
+## Post-Training Optimization Algorithms
+
+|Compression algorithm|PyTorch|TensorFlow|   ONNX    |OpenVINO|
+|:--------------------| :---: | :---: |:---------:| :---: |
+| Quantization        | Supported | Supported | Supported | Supported |
```

Comment:
Have we tested `PyTorch` & `TensorFlow` with Post-Training Optimization pipeline? Are all the algorithms supporting these frameworks?

Reply:
I also wonder this. What I found out that we declare the examples with PTQ for Torch and TF without experimental mark. Moreover, We don't have any log messages that this is an experimental feature for Torch and TF

Reply:
It looks like our omission that we have no messages about it.

Reply:
Remove this table. Only vouch for the ONNX in the scope of this PR.

Reply:
> Remove this table. Only vouch for the ONNX in the scope of this PR.

I plan to keep this table but mark all backends except ONNX as an experimental as we have samples and ready to use functional support.
What do you think?

| Compression algorithm                                                       |         ONNX          |TensorFlow|         PyTorch          |OpenVINO|
|:----------------------------------------------------------------------------|:------------------------:| :---: |:------------------------:| :---: |
| [Quantization](./docs/compression_algorithms/post_training/Quantization.md) | Supported  | Supported (Experimental)| Supported (Experimental) | Supported (Experimental)|

Reply:
A table with only one data row looks odd. Just add a regular sentence describing the current situation.

Reply:
But the table is much easier to understand. I would stand on it.